Stylesheets/components/avtar.css

```diff
@@ -0,0 +1,76 @@
+@import url("/stylesheets/main.css");
+
+.center {
+  margin-left: 0%;
+}
+
+.avtar {
+  margin: 0.3rem;
+  width: 2rem;
+  height: 2rem;
+  border-radius: 50%;
+  position: relative;
+  left: 30%;
+}
+
+.avtar-small {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.avtar-large {
+  width: 3rem;
+  height: 3rem;
+}
+
+.avtar-xlarge {
+  width: 4.5rem;
+  height: 4.5rem;
+}
+
+.avtar-xlarger {
```

Comment:
A small suggestion in the class naming here xlarger and large name seems a bit similar you can give the name as xxlarge which is easier for user to use

Reply:
okay! I did it.